protocol/xmrmaker/net.go

```diff
@@ -33,7 +33,7 @@ func (b *Instance) initiate(
 	}
 
 	// check user's balance and that they actually have what they will provide
-	if balance.UnlockedBalance <= float64(providesAmount) {
+	if balance.UnlockedBalance <= uint64(providesAmount) {
```

Comment:
what units is the `balance.UnlockedBalance` in now? is it piconero? if so we need to change this check so that the units match

Reply:
`balance.UnlockedBalance` is piconero.  `providesAmount` is `common.MoneroAmount` which, to my understanding, is also piconero.